documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -406,6 +406,201 @@ To obtain a list of caches available in a {brandname} cluster, invoke a `GET` re
 include::rest_examples/get_v2_list_caches.adoc[]
 ----
 
+[[rest_v2_cache_xsite]]
+=== Cross site replication
+
+Use the REST API to monitor and control Cross Site (x-site) replication on your {brandname} cluster.
+
+==== Getting status of all backup sites
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/
+----
+
+The response contains each site followed by a description of the status.
+Example:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+{
+  "NYC": "online",
+  "LON": "offline"
+}
+----
+
+.Returned Status
+|===
+|Value |Description
+
+|`online`
+|All nodes in the backup site are online
+
+|`offline`
+|All node in the backup site are offline
+
+|`mixed`
+|Some nodes in the backup site are online and others offline. It will include in the status the nodes that are offline. E.g.: `mixed, offline on nodes: Node1, Node2`
+|===
+
+==== Getting status of a backup site
+
+To obtain the status of a single backup site:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{siteName}
+----
+
+The response contains each node in the backup site with the status.
+Example:
+
+[source,json,options="nowrap",subs=attributes+]
+----
+{
+  "NodeA":"offline",
+  "NodeB":"online"
+}
+----
+
+.Returned Status
+|===
+|Value |Description
+
+|`online`
+|The node is online
+
+|`offline`
+|The node is offline
+
+|`failed`
+|Failed to obtain status, the remote cache could be shutting down or a network error occurred during the request
+|===
+
+==== Taking a backup site offline
+
+To take a backup site `backName` offline, for the cache `cacheName`, execute a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /v2/caches/{cacheName}/x-site/backups/{backupName}?action=take-offline
```

Comment:
some places have `backupName`, other places `site` or `siteName`.
Can't we use `siteName` everywhere?